Assets/Scripts/Player/ResourceManager.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ResourceManager : MonoBehaviour
+{
+    
+    public List<Resource> resources;
+    
+    void Awake()
+    {
+        foreach(Resource resource in resources)
+        {
+            resource.Init();
+        }
+    }
+
+}
```

Comment:
Nice - hard to tell in the diff, is this component also added to the `MainGameSystems` prefab? That way it will automagically exist in every scene

Reply:
Oh nevermind, the "Manager" part of the name threw me off. In my mind, managers are top-level systems.

Reply:
Actually, this might not work. We don't want a resource to be initialized everytime the player is instantiated, right? It may make sense to move this to a top-level system so that Init only fires once when the game starts.